resources/views/profile/index.blade.php

```diff
@@ -120,6 +120,121 @@
                     </div>
                 </div>
             </div>
+            @if(setting('shop.use-site-money'))
```

Comment:
This should be a standalone setting

Reply:
Since the money transfer relies on the shop, maybe adding a setting shop.allow_users_money_transfer would be better suited than shop.use-site-money indeed

And the line would then be `@if(setting('shop.use-site-money') && setting('shop.allow_users_money_transfer') )`

Reply:
> Since the money transfer relies on the shop
Currently only the shop plugin use it, but it's a generic feature that could be use by any plugins